src/lang/en-US.ts

```diff
@@ -107,4 +90,31 @@ const enUSMessages: ResolvedIntlConfig['messages'] = {
     'changeSet.data.details': `Details`,
 };
 
+const Capture_Create: ResolvedIntlConfig['messages'] = {
+    'captureCreation.ctas.discover': `Test Config`,
+    'captureCreation.heading': `New Capture`,
+    'captureCreation.instructions': `To get started please provide a unique name and the source type of the Capture you want to create. Once you've filled out the source details you can click "Test Capture" down below to test the connection.`,
+    'captureCreation.missingConnectors': `No connectors installed. You must install a source connector before being able to create a capture.`,
+    'captureCreation.tenant.label': `Tenant`,
+    'captureCreation.name.label': `Name`,
+    'captureCreation.name.description': `Name of the capture - must be unique. (ex: acemCo/marketing_data)`,
+    'captureCreation.image.label': `Source`,
+    'captureCreation.image.description': `The connector you want to use to connect to your endpoint.`,
+    'captureCreation.config.source.doclink': `Connector Docs`,
+    'captureCreation.save.failed': `Capture creation failed. Please see below for details:`,
+    'captureCreation.editor.default': `Before you can edit the Capture Catalog you need to fill out the connection config section and click Discover`,
+    'captureCreation.finalReview.instructions': `Look over the catalog configuration that was generated. If you want to edit anything you can do that directly in the editor. Once you're ready you can download the file for your local.`,
+};
+
+const Materialization_Create: ResolvedIntlConfig['messages'] = {
+    'materializationCreation.heading': `New Materialization`,
+    'materializationCreation.instructions': `To get started please provide a unique name, select an endpoint type, and select the Collections you want to Materialize. After that you can review the YAML before saving.`,
+};
+
+const enUSMessages: ResolvedIntlConfig['messages'] = {
+    ...CommonMessages,
+    ...Capture_Create,
+    ...Materialization_Create,
+};
```

Comment:
This is something we've talked about before. Trying it out on some of the pages. If we like it we can use it everywhere. This does allow us to easily reference common messages and stuff like that.